mirage/config.ts

```diff
@@ -318,6 +318,15 @@ export default function(this: Server) {
     });
     this.post('/search/collections/', searchCollections);
 
+    osfResource(this, 'custom-file-metadata-record', {
+        only: ['show', 'update'],
+        path: '/custom-file-metadata-records',
```

Comment:
In the urls, our API usually use snake case. Are we sure the path is not `/custom_file_metadata_records`?